src/plugin/featureaction/ui/editfeatureaction.js

```diff
@@ -111,6 +115,45 @@ goog.exportProperty(
     'showActionPreview',
     plugin.im.action.feature.ui.EditFeatureActionCtrl.prototype.showActionPreview);
 
+/**
+ * Create the canvas preview for a selected sound
+ * @param {plugin.im.action.feature.SoundAction} soundAction
+ */
+plugin.im.action.feature.ui.EditFeatureActionCtrl.prototype.buildSoundPreview = function(soundAction) {
+  this.soundCanvas = /** @type {HTMLCanvasElement} */ (document.createElement(
```

Comment:
Is this writing the sound name to a canvas? This would be much more easily done with a DOM element like `span` or `div`. I would suggest using a font awesome icon to indicate that it's a sound, along with text indicating the sound name. Example:
```
<span><i class="fa fa-music"></i> (sound name)</span>
```

Reply:
I was playing around with the icon and to me, it seemed like to cluttered the space and looked a little awkward. So I left it out , but if you think it would work better I can easily add it back in. I did add the span instead of using the canvas!